File: Java/src/LinkedListCycleIITest.java
import org.junit.Assert;
import org.junit.Test;

public class LinkedListCycleIITest {

    @Test
    public void testDetectCycle() {
        LinkedListCycleII l = new LinkedListCycleII();

        Assert.assertNull(l.detectCycle(null));

        Assert.assertNull(l.detectCycle(new LinkedListCycleII.ListNode(0)));
    }

}

File: Kotlin/src/LinkedListCycleIITest.kt
import org.junit.Assert
import org.junit.Test

class LinkedListCycleIITest {

    @Test
    fun testDetectCycle() {
        val l = LinkedListCycleII()

        Assert.assertNull(l.detectCycle(null))

        Assert.assertNull(l.detectCycle(LinkedListCycleII.ListNode(0)))
    }

}